CRM/CiviAwards/BAO/AwardDetail.php

```diff
@@ -76,4 +77,34 @@ private static function validateDates(array $params) {
     }
   }
 
+
+  /**
+   * Validates review_field options to ensure legal options are passed.
+   *
+   * @param array $params
+   *  Request Parameters
+   *
+   * @throws \Exception
+   */
+  private static function validateReviewFields(array $params) {
+    $reviewFieldsLegalOptions = ['id', 'weight', 'required'];
+    $reviewFieldsOptions = CRM_Utils_Array::value('review_fields', $params);
+    if (!$reviewFieldsOptions) {
+      throw new Exception(ts('Valid review fields options must be set'));
```

Comment:
We should not throw an exception if the review fields are not sent. The Award details UI is structured in such a way that basic fields are added on create and then, other fields can be added on edit. If you see here: https://github.com/compucorp/uk.co.compucorp.civiawards/blob/RSE-44-awards-management/CRM/CiviAwards/Service/AwardDetail.php#L164, it shows that the review fields can actually be empty.